test/import-map.test.js

```diff
@@ -0,0 +1,106 @@
+const request = require("supertest");
+const { app, setConfig } = require("../src/web-server");
+
+beforeAll(() => {
+  setConfig({
+    manifestFormat: "importmap",
+    username: "username",
+    password: "password",
+    locations: {
+      prod: "memory://prod",
+    },
+  });
+});
+
+describe(`/import-map.json`, () => {
+  it(`does not return anything when it's not setup yet.`, async () => {
+    const healthResponse = await request(app)
```

Comment:
This is not a response to the health endpoint:

```suggestion
    const response = await request(app)
```